src/components/backend-ai-scaling-group-list.ts

```diff
@@ -500,6 +678,15 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
             label="${_t('resourceGroup.WsproxyAddress')}"
             placeholder="http://localhost:10200"
           ></mwc-textfield>
```

Comment:
I'm not sure about using "nested dialog" here. Could you explain the reason for this layout?

Reply:
Yes, 
 As the scheduler option is added, a new input form for the option should be added, and if these input forms are not separated through dialog, the length of the create/modify dialog will become longer and I think less readable each time the option is added.

Reply:
Hmm, but the number of keys may not exceed 5 or 6, so I would like to suggest accordion UI instead of nested dialog.

Reply:
I revise it following comment.

**image**
![PR6_review2_2](https://user-images.githubusercontent.com/43121372/154313524-2501596a-2142-45bd-967f-a934d3f699d8.png)